finger-exercises/redux/src/app/screens/App/components/ShoppingCart/index.js

```diff
@@ -1,20 +1,25 @@
-import React, { PureComponent, Fragment } from 'react';
-import { arrayOf, func } from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import { bookSelectedPropType } from '@constants/propTypes';
+import shopingCartActions from '@redux/shoppingCart/actions';
 import Button from '@components/Button';
 
 import Item from './components/Item';
 import styles from './styles.scss';
 
-class ShoppingCart extends PureComponent {
-  state = {
-    open: false
-  };
+class ShoppingCart extends Component {
+  componentDidMount() {
+    this.props.openChart();
```

Comment:
I'm not sure about this? 
As it is, the cart (icon + box) is mounted when a book is "selected" and the actual cart gets opened by this call.
Wouldn't we want the cart (icon) always mounted but just show the cart box on toggle?

Reply:
Can we see it on Friday? Because I dont know how to do it